tests/test_ote_api.py

```diff
@@ -51,27 +52,36 @@
 from mmdet.apis.ote.apis.detection.config_utils import set_values_as_default
 from mmdet.apis.ote.apis.detection.ote_utils import generate_label_schema
 
+from e2e_test_system import e2e_pytest_api
```

Comment:
Just wondering if that kind of import works? From what directory should the tests be run?

Reply:
Sorry, I do not know. But it works!